setup.py

```diff
@@ -0,0 +1,41 @@
+"""Setup for html XBlock."""
+
+import os
+
+from setuptools import setup
+
+
+def package_data(pkg, roots):
+    """Generic function to find package_data.
+
+    All of the files under each of the `roots` will be declared as package
+    data for package `pkg`.
+
+    """
+    data = []
+    for root in roots:
+        for dirname, _, files in os.walk(os.path.join(pkg, root)):
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
+
+    return {pkg: data}
+
+
+setup(
+    name='html-xblock',
+    version='0.1',
+    description='HTML XBlock will help creating and using a secure and easy-to-use HTML blocks',
+    license='AGPL v3',
+    packages=[
+        'html',
+    ],
+    install_requires=[
+        'XBlock',
+    ],
+    entry_points={
+        'xblock.v1': [
+            'html = html:HTMLXBlock',
```

Comment:
Since this XBlock won't be backwards compatible (in some cases), we'll need to use a different entry point other than `html`.

Reply:
So you suggest to rename the `html` before the assignment to not conflict with other XBlock or there's something I'm missing?

Reply:
So you suggest to rename the `html` before the assignment to not conflict with other `XBlock`s or there's something I'm missing?

Reply:
I suggest using a different entry point like `html5` for this XBlock. So all the existing HTML XModules in all courses will be unchanged and will keep using the old XModule (so that we don't break anything), but new HTML components added to courses will use this new XBlock.